### PR TITLE
try.mongodb.org now redirects to university.mongodb.com

### PR DESCRIPTION
**Pull request by gianpaj**

### CODE REVIEW

File: docs/landing/layouts/partials/header/topRight.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div>
 <div class="nav-items pull-right">
-  <a href="http://try.mongodb.org/" data-toggle="tooltip" data-placement="bottom" title="Try MongoDB in the browser">Try it Out</a>
+  <a href="https://university.mongodb.com" data-toggle="tooltip" data-placement="bottom" title="Free Online Classes">MongoDB University</a>
```
Comment (evanchooly) [inline docs/landing/layouts/partials/header/topRight.html]: probably also needs to be done in reference/

**Comment by gianpaj** [comment]
thanks. my bad

**Comment by evanchooly** [comment]
I meant you'll need to do it in both places since we define that header twice.

**Comment by gianpaj** [comment]
Oh sorry (again). didn't know there were two places. I've made a second commit on #312 - hope that's ok. happy to do one commit instead

**Pull request closed without merging**